neurodamus/utils/memory.py

```diff
@@ -198,25 +198,29 @@ def print_allocation_stats(rank_allocation, rank_memory):
 
 
 @run_only_rank0
-def export_allocation_stats(rank_allocation, filename):
+def export_allocation_stats(rank_allocation, filename, metype_memory_usage, memory_per_metype_file):
```

Comment:
I would separate the two for a simple reason: An allocation can be generated anytime from `metype_memory_usage` and `memory_per_metype_file`, so we may want to store the parts independently

Reply:
Makes sense, will split (also rename a few variables because like this it's extremely unclear).